components/codec/src/number.rs

```diff
@@ -919,6 +941,17 @@ pub trait NumberEncoder: BufferWriter {
         write!(self, v, 4, encode_i32_le)
     }
 
+    /// Writes a 32 bit float number `v` in little endian,
```

Comment:
For completeness, maybe providing a `read_f32_le` is better (although you may not need it).

Reply:
Update.